massinstall/massinstall.go

```diff
@@ -130,27 +130,41 @@ func (mi *MassInstall) Run(md *model.SystemInstall, rootDir string, options args
 	// If there are no media defined, then we should look for
 	// Advanced Configuration labels
 	if len(md.TargetMedias) > 0 {
+		// If the partitions are defined from the configuration file,
+		// assume the user knows what they are doing and ignore validation checks
+		if !options.SkipValidationSizeSet && !options.SkipValidationAllSet {
+			md.SkipValidationSize = true
+			md.SkipValidationAll = true
+		} else {
+			if !options.SkipValidationSizeSet {
+				md.SkipValidationSize = true
+			} else {
+				if !options.SkipValidationAllSet {
+					md.SkipValidationAll = false
+				}
+			}
+		}
+
 		// Need to ensure the partitioner knows we are running from
 		// the command line and will be using the whole disk
 		for _, curr := range md.TargetMedias {
 			md.InstallSelected[curr.Name] = storage.InstallTarget{WholeDisk: true}
 			log.Debug("Mass installer using defined media in YAML")
 		}
 
-		if md.SkipValidationAll {
-			log.Warning("massinstall: Skipping all partition checks due to SkipValidationAll")
+		if md.IsDesktopInstall() {
```

Comment:
Same here...Why perform IsDesktopInstall check?

Reply:
IsDesktopInstall looks at the list of bundles to be installed. If there are desktop related bundles, we need much more disk space.
This is why there is a check for Desktop or Server, as that impacts the minimum required disk space needed for the installation regardless if we are installing from the CLI, TUI, or GUI.

Reply:
Same as above